dockerfiles/hack/run-integration.py

```diff
@@ -55,7 +55,7 @@ def build_entry_point_args(command: click.Command, config: str,
                            dry_run: Optional[str], integration_name: str,
                            extra_args: Optional[str]) -> list[str]:
     args = ['--config', config]
-    if dry_run is not None:
+    if dry_run in ['--dry-run', '--no-dry-run']:
```

Comment:
Does this change belong in this PR?

Reply:
Was hoping it could :) 
But I will create a separate PR for this now :+1:

Reply:
Fixed :+1: 

Created new commits now to make review easier -> will squash after review

Reply:
The dry-run change lives here now :)
https://github.com/app-sre/qontract-reconcile/pull/2119